R/lint.R

```diff
@@ -125,64 +125,53 @@ reorder_lints <- function(lints) {
     ]
 }
 
-#' Lint a package
+#' Lint a directory
 #'
-#' Apply one or more linters to all of the R files in a package.
-#' @param path the path to the base directory of the package, if \code{NULL},
+#' Apply one or more linters to all of the R files in a directory
+#' @param path the path to the base directory, if \code{NULL},
```

Comment:
Could you check what actually happens when NULL is passed as the base-path, from the code it looks like the function will die (because `dir` will)